northbound/impl/src/main/java/org/opendaylight/ansible/northbound/IetfL3vpnSvcServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2018 Red Hat, Inc. and others. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.opendaylight.ansible.northbound;
+
+import static org.opendaylight.ansible.mdsalutils.Datastore.OPERATIONAL;
+import static org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.l3vpn.svc.rev170502.Status.InProgress;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.opendaylight.ansible.mdsalutils.RetryingManagedNewTransactionRunner;
+import org.opendaylight.controller.md.sal.binding.api.DataBroker;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.l3vpn.svc.rev170502.CommitL3vpnSvcInput;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.l3vpn.svc.rev170502.CommitL3vpnSvcOutput;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.l3vpn.svc.rev170502.CommitL3vpnSvcOutputBuilder;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.l3vpn.svc.rev170502.IetfL3vpnSvcService;
+import org.opendaylight.yangtools.yang.common.RpcResult;
+import org.opendaylight.yangtools.yang.common.RpcResultBuilder;
+import org.ops4j.pax.cdi.api.OsgiService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class IetfL3vpnSvcServiceImpl implements IetfL3vpnSvcService {
+    private static final Logger LOG = LoggerFactory.getLogger(IetfL3vpnSvcServiceImpl.class);
+    private final RetryingManagedNewTransactionRunner txRunner;
+    private static final String COMMIT_VERSION = "123";
```

Comment:
what is this?

Reply:
COMMIT_VERSION? that is what the api returns. It maps to the currently configured l3vpn. The api looks to only have a single config at a time. Push config, then commit - return a version string is response to the commit. I chose 123 as a version since the demo only has one. If we add the ability to delete and create another we will update version to increment.